pokedex/pokedex/settings.py

```diff
@@ -133,4 +133,6 @@
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-CORS_ORIGIN_ALLOW_ALL = True
\ No newline at end of file
+CORS_ORIGIN_ALLOW_ALL = True
+
+USE_TZ = False
```

Comment:
this is to get rid of an annoying time zone warning that keeps showing up on the backend terminal. Since we don't really care about time zones, we're just removing this

Reply:
nice